internal/cri/config/streaming.go

```diff
@@ -40,21 +40,21 @@ const (
 
 func getStreamListenerMode(config *ServerConfig) (streamListenerMode, error) {
 	if config.EnableTLSStreaming {
-		if config.X509KeyPairStreaming.TLSCertFile != "" && config.X509KeyPairStreaming.TLSKeyFile != "" {
+		if config.TLSCertFile != "" && config.TLSKeyFile != "" {
```

Comment:
See my other comment on the `QF1008` "quick fix"; I think we should consider disabling it, as sometimes explicit > implicit; also to reduce unneeded code-churn.